### PR TITLE
kvm-snapshot-explanation

### PR DESCRIPTION
**Pull request by andrijapanicsb**

### CODE REVIEW

**Comment by andrijapanicsb** [comment]
requesting docbuild

File: source/adminguide/storage.rst [hunk 1]
```diff
@@ -679,6 +679,19 @@ How to Snapshot a Volume
 
 #. Click the Snapshot button. |SnapshotButton.png|
 
+KVM volume Snapshot specifics
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In recent CloudStack versions, by default, creating a volume snapshot for a running VM is disabled
+due to a possible volume corruption in certain cases. To enable creating a volume snapshots while the VM
+is running, the global setting 'kvm.snapshot.enabled' must be set to 'True'.
```
Comment (nvazquez) [inline source/adminguide/storage.rst:687]:
```suggestion
is running, the global setting 'kvm.snapshot.enabled' must be set to 'True'.

```

File: source/adminguide/storage.rst [hunk 2]
```diff
@@ -679,6 +679,19 @@ How to Snapshot a Volume
...
+The way the volume snapshot is now created has been changed.
```
Comment (nvazquez) [inline source/adminguide/storage.rst:688]:
```suggestion
The volume snapshot creation has changed in recent versions:

```

File: source/adminguide/storage.rst (hunk 1 above)
Comment (andrijapanicsb) [reply source/adminguide/storage.rst:687]: I don't see any differences here?

**Comment by andrijapanicsb** [comment]
I wanna see docbuild

**Comment by andrijapanicsb** [comment]
requesting docbuild

**Pull request merged**